src/hooks/usePageReducer.js

```diff
@@ -66,14 +66,16 @@ export const usePageReducer = (
   const getState = () => stateRef.current;
 
   // Basic dispatch function.
-  const dispatch = action => {
+  const dispatch = useCallback(action => {
     const newState = DataTablePageReducer(getState(), action);
     setPageState(newState);
     stateRef.current = newState;
-  };
+  });
```

Comment:
should this have an array?

```suggestion
  }, []);
```

Reply:
yup